frameworks/hdfs/universe/config.json

```diff
@@ -778,6 +788,11 @@
           "media": {
             "type": "application/x-zone-constraints+json"
           }
+        },
+        "hadoop_datanode_opts": {
```

Comment:
The descriptions for these three options could be improved a bit -- including how they interact with the `hadoop_opts` parameter.

Reply:
👍 added a bit more detail to the descriptions